src/components/ContactSection/ContactSection.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import SectionHeader from '../../styles/elements/SectionHeader';
```

Comment:
I haven;t check without typescript, but I suggest to think about absolute paths right now, not later

Reply:
added as a TODO comment